mllib/src/main/scala/org/apache/spark/mllib/classification/NaiveBayes.scala

```diff
@@ -41,6 +44,10 @@ class NaiveBayesModel private[mllib] (
     val pi: Array[Double],
     val theta: Array[Array[Double]]) extends ClassificationModel with Serializable with Saveable {
 
+  /** A Java-friendly constructor that takes three Iterable parameters. */
+  def this(labels: JIterable[Double], pi: JIterable[Double], theta: JIterable[JIterable[Double]]) =
```

Comment:
Maybe we should mark this as package private because the only use case would be calling from python.